runner/webserver.ts

```diff
@@ -142,12 +143,13 @@ export function webserver(wct: Context): void {
     // webserver as they please.
     await wct.emitHook('prepare:webserver', app);
 
-    // Serve up all the static assets.
-    app.use(serveWaterfall(wsOptions.pathMappings, {
+    // Serve up project & dependencies via polyserve
+    const polyserve = makeApp({
       root: options.root,
       headers: DEFAULT_HEADERS,
-      log: wct.emit.bind(wct, 'log:debug'),
-    }));
+      packageName: path.basename(options.root),
```

Comment:
packageName is normally read from `bower.json`. This matters in cases where there's a cycle in the dependency graph that points back to the project under test.